programs/airdrop/src/common.rs

```diff
@@ -15,6 +15,14 @@ pub struct State {
     pub vault: Pubkey,
     pub vault_bump: u8,
 
+    /// Seed and bump for this account. Should not be needed, but saved for
```

Comment:
Is the comment correct? These are needed now for realms?

Reply:
This comment is correct. We dont need to store them, but will make it a ton easier if in the future we ever need to make the state mutable.